CMakeLists.txt

```diff
@@ -149,7 +149,7 @@ endif()
 
 if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX OR CMAKE_COMPILER_IS_CLANG)
 	set(CMAKE_CXX_FLAGS "-Wall -Wextra -Wvla -Wno-unused-function -Wno-missing-field-initializers ${CMAKE_CXX_FLAGS} -fno-strict-aliasing")
-	set(CMAKE_C_FLAGS "-Wall -Wextra -Wvla -Wno-unused-function -Werror-implicit-function-declaration -Wno-missing-braces -Wno-missing-field-initializers ${CMAKE_C_FLAGS} -std=gnu99 -fno-strict-aliasing")
+	set(CMAKE_C_FLAGS "-Wall -Wextra -Wvla -Wno-unused-function -Werror-implicit-function-declaration -Wno-missing-braces -Wno-missing-field-initializers -lpci ${CMAKE_C_FLAGS} -std=gnu99 -fno-strict-aliasing")
```

Comment:
This is not the appropriate way to add libraries and you have added it to the wrong binary.

See the plugins/obs-ffmpeg/CMakeLists.txt for how to add libraries to the binary you are modifying. If you wanted to use libpci you will probably also need to create a FindPci.cmake module.

Reply:
Thanks to tytan652's comment I should have fixed it. Let's see how the CI goes!